podcasts/ExpandedCollectionViewController+CollectionView.swift

```diff
@@ -48,8 +48,10 @@ extension ExpandedCollectionViewController: UICollectionViewDataSource, UICollec
 
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
         guard podcastCollection != nil else { return CGSize.zero }
-        let indexPath = IndexPath(row: 0, section: section)
-        let headerView = self.collectionView(collectionView, viewForSupplementaryElementOfKind: UICollectionView.elementKindSectionHeader, at: indexPath)
+
+        let headerView: DiscoverCollectionHeader = DiscoverCollectionHeader.fromNib()
+        headerView.populate(podcastCollection: podcastCollection)
+        headerView.linkDelegate = self
```

Comment:
What do you think about removing the delegate here? I guess it's not necessary

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2206/commits/5f9a0c89d6f2c563a0c8ef77a46886b0099052ad